components/suggestion/suggestion_list.jsx

```diff
@@ -11,11 +11,13 @@ import {Constants} from 'utils/constants.jsx';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import LoadingSpinner from 'components/widgets/loading/loading_spinner';
 
+// When this file is migrated to TypeScript, type definitions for its props already exist in ./suggestion_list.d.ts.
```

Comment:
Why not migrate this along this change?

Reply:
I forget exactly why I didn't migrate this when I originally did this a few weeks back, but I think there was just too much going on between this and the other related components to migrate it over.